src/server/util/dogstatsd.ts

```diff
@@ -6,6 +6,8 @@ export const API_KEY_GENERATE_TAG_IS_NEW = 'isnew'
 export const ERROR_UNHANDLED_REJECTION = 'error.unhandled_rejection'
 export const MALICIOUS_ACTIVITY_FILE = 'malicious_activity.file'
 export const MALICIOUS_ACTIVITY_LINK = 'malicious_activity.link'
+export const SCAN_FAILED_FILE = 'scan.file.failure'
+export const SCAN_FAILED_LINK = 'scan.link.failure'
```

Comment:
nit: maybe can list in alphabetical order? but this grouping makes sense so it's okay too

Reply:
fixed [0266848](https://github.com/opengovsg/GoGovSG/pull/2145/commits/0266848ff31e067efcc5ca88d934e07575a7ed99)